src/geo/cartodb-layer-group.js

```diff
@@ -46,17 +46,34 @@ var CartoDBLayerGroup = Backbone.Model.extend({
     return !!this.get('urls');
   },
 
-  getTileURLTemplates: function (type) {
+  getSubdomains: function () {
+    return this.get('urls') && this.get('urls').subdomains;
+  },
+
+  getTileURLGrid: function () {
```

Comment:
This method is basically returning an array of urlTemplates with the `{s}` replaced by the subdomain... right? I don't quite understand the name... What about merging this method with getTileURLTemplate so we can do something like:

```
> model.getTileURLTemplates();
[
  "http://{s}.ashbu.cartocdn.com/documentation/api/v1/map/010b3e5e4b0e9997024b739bb538260a:0/0/{z}/{x}/{y}.png
]
> model.getTileURLTemplates({ replaceSubomains: true });
[
  "http://1.ashbu.cartocdn.com/documentation/api/v1/map/010b3e5e4b0e9997024b739bb538260a:0/0/{z}/{x}/{y}.png",
  "http://2.ashbu.cartocdn.com/documentation/api/v1/map/010b3e5e4b0e9997024b739bb538260a:0/0/{z}/{x}/{y}.png",
  "http://3.ashbu.cartocdn.com/documentation/api/v1/map/010b3e5e4b0e9997024b739bb538260a:0/0/{z}/{x}/{y}.png",
  "http://4.ashbu.cartocdn.com/documentation/api/v1/map/010b3e5e4b0e9997024b739bb538260a:0/0/{z}/{x}/{y}.png"
]

```